pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1801,7 +1801,7 @@
       "BaselineVersion": "4.0.0",
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.0.0",
-        "4.0.1.0": "4.3.0"
+        "4.1.0.0": "4.3.0"
```

Comment:
And since the 4.0.1.0 entry is already on a release branch, I don't know if we should be deleting that line?